docs/customize/users.md

```diff
@@ -0,0 +1,219 @@
+# Adding Users to Invenio ILS instance
```

Comment:
We branded the products InvenioILS and InvenioRDM without spaces:
```suggestion
# Adding users to InvenioILS instance
```
Can you change it everywhere?